docs/devsetup.md

```diff
@@ -1,7 +1,22 @@
 ---
-id: devsetup
-title: Dev Setup
-sidebar_label: Dev Setup
+id: devsetup 
+title: Getting Started for Developers 
+sidebar_label: For Developers 
 ---
 
-paste old setups here
+
+1. `git clone https://github.com/webmixedreality/exokit.git`
```

Comment:
Since these three are filler instructions, maybe we should fill in something other than copypasta?
We wouldn’t want to increase the level of confusion in the docs.

Reply:
Just noticed the formatting is off anyways with userappsetup.md and usertabsetup.md, going to change and integrate the get.webmr.io way of installation for app/reality tab quickstarts

Reply:
Makes sense. Should definitely be a fill information flow for all of the cases. :+1:

Reply:
0a9a39f

Reply:
and 
a1f0751  updated info flow as well